File: PHP/account.php
<?php
class Account {
    public $id;
    public $name;
    public $document;
    public $email;
    public $password;
    public $UserType;

    public function __construct($id, $name, $document, $email, $password, $userType){
        $this->id = $id;
        $this->name = $name;
        $this->document = $document;
        $this->email = $email;
        $this->password = $password;
        $this->userType = $userType;
    }
    public function PrintDataPaymentAccount(){
        echo "Id: $this->id; Name: $this->name; Document: $this->document; Email; $this->email; Password; $this->password; User type; $this->userType <br/>";
    }
}

File: Java/Account.java
class Account {
    Integer id;
    String name;
    String document;
    String email;
    String password;

    public Account(Integer id, String name, String document, String email, String password){
        this.id = id;
        this.name = name;
        this.document = document;
        this.email = email;
        this.password = password;
    }
    void printDataAccount() {
        System.out.println("Id: "+ id + "| Name: " + name + "| Document: " + document + "| Email: " + email + "| Password: " + password);
    }
}